app/install.sh.tmpl

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
This template file is copied from `cmd/geninstaller/install.sh.tmpl`, and is bit-for-bit identical to the latter.